### ISSUE TITLE
initialize is passing wrong parameters to action

### ISSUE THREAD

**Issue by thejohnbackes** [erikras/redux-form#2294]
I have a form that calls this method: 
`initialize('ruleSearch', {}, false);`
The action that is created and dispatched is:
```{
  type: '@@redux-form/INITIALIZE',
  meta: {
    form: 'ruleSearch',
    keepDirty: {}
  },
  payload: 'ruleSearch'
}```
Debugging yields the following:
```function bindActionCreator(actionCreator, dispatch) {
  return function () {
    return dispatch(actionCreator.apply(undefined, arguments));
  };
}
// arguments === [ 'ruleSearch', {}, false ];    // looks good!
// actionCreator.name === "initialize"
```
then step into actionCreator:

```
var initialize = exports.initialize = function initialize(form, values, keepDirty) {
  return { type: _actionTypes.INITIALIZE, meta: { form: form, keepDirty: keepDirty }, payload: values };
};
// form === 'ruleSearch'; // good!
// values === 'ruleSearch'; // uh oh!
// keepDirty === {}; // not. good.
// arguments === ["ruleSearch", "ruleSearch", {}, false]
```
So somehow an extra argument is getting added between bindActionCreators and initialize()!

I'm out of ideas and my brain is full. Any thoughts?

**Comment by erikras** [comment]
You are confusing "_already bound to your form *and* to `dispatch`"_ [`this.props.initialize(values)`](http://redux-form.com/6.3.1/docs/api/Props.md/#-initialize-data-object-function-) **action** function  with the "not bound to anything – use at your own peril with `dispatch()`" [`initialize(form, values, keepDirty)`](-initialize-form-string-data-object-keepdirty-boolean-) **action creator**.

TL;DR: You should just be calling `this.props.initialize({})`.

**Comment by thejohnbackes** [comment]
hah, I realized that on the drive home. DOH! Thanks!
